spec/system/ai_bot/artifact_spec.rb

```diff
@@ -48,30 +48,26 @@
     find(".ai-artifact__click-to-run button").click
 
     artifact_element_selector = ".ai-artifact[data-ai-artifact-id='#{ai_artifact.id}']"
-    iframe_selector = "#{artifact_element_selector} iframe"
+    artifact_element = find(artifact_element_selector)
 
-    expect(page).to have_css(iframe_selector)
```

Comment:
We are doing a find at tnetx step so I don't think that's necessary, we will fail on the find if it doesn’t exist